packages/cliqz-home-react/src/Cliqz.js

```diff
@@ -19,12 +20,32 @@ class Cliqz  {
   constructor(props) {
     const freshtab = createSpananForModule('freshtab');
     const core = createSpananForModule('core');
-
     window.addEventListener('message', event => {
```

Comment:
please take a look on recent development of spanan. The export api should help to deal with this scenario https://github.com/chrmod/spanan